ioctl/newcmd/config/config.go

```diff
@@ -128,6 +128,42 @@ func (c *info) reset() error {
 	return nil
 }
 
+// get retrieves a config item from its key.
+func (c *info) get(arg string) (string, error) {
+	switch arg {
+	case "endpoint":
+		if c.readConfig.Endpoint == "" {
+			return "", config.ErrEmptyEndpoint
+		}
+		message := endpointMessage{Endpoint: c.readConfig.Endpoint, SecureConnect: c.readConfig.SecureConnect}
+		return fmt.Sprint(message.Endpoint, "    secure connect(TLS):", message.SecureConnect), nil
+	case "wallet":
+		return c.readConfig.Wallet, nil
+	case "defaultacc":
+		if c.readConfig.DefaultAccount.AddressOrAlias == "" {
+			return "", config.ErrConfigDefaultAccountNotSet
+		}
+		return c.readConfig.DefaultAccount.String(), nil
+	case "explorer":
+		return c.readConfig.Explorer, nil
+	case "language":
+		return c.readConfig.Language, nil
+	case "nsv2height":
+		return strconv.FormatUint(c.readConfig.Nsv2height, 10), nil
+	case "analyserEndpoint":
+		return c.readConfig.AnalyserEndpoint, nil
+	case "all":
+		return c.readConfig.String(), nil
```

Comment:
line146 and line156  calls output's function.  https://github.com/iotexproject/iotex-core/blob/2ed5b0387338860ed93331c19f9dc589f21da94e/ioctl/output/format.go#L180
directly refactor it here and call.  don't use output

Reply:
Thanks for your help, done.